src/animation/AnimationMixer.js

```diff
@@ -83,8 +83,10 @@ Object.assign( THREE.AnimationMixer.prototype, THREE.EventDispatcher.prototype,
 
 		var root = optionalRoot || this._root,
 			rootUuid = root.uuid,
-			clipName = ( typeof clip === 'string' ) ? clip : clip.name,
-			actionsForClip = this._actionsByClip[ clipName ];
+			clipObject = typeof clip === 'string' ? this.getClipByName( clip ) : clip,
```

Comment:
@bhouston 
> I am okay if we continue down this road, but be aware that we will need to eventually create a new per-character mixer class that makes using this global mixer easier for individual characters.

This might be something @sunag is actually building.

Considering the current functionality, and that most of it sits on the actions anyway, a per-character mixer would be a trivial wrapper binding a default for `localRoot`. Am I missing something?